app/services/evacancy/ev_connect_service.rb

```diff
@@ -0,0 +1,64 @@
+module Evacancy
+  class EvConnectService
+    class << self
+      def check_ports
+        token = check_access_token
+
+        previously_open = 0
+        currently_open = 0
+
+        EvConnectStationPort.all.each do |station|
+          new_status = get_status(token, station.qr_code)
+          previously_open += 1 if station.port_status == 'AVAILABLE'
+          currently_open += 1 if new_status == 'AVAILABLE'
+          station.update(port_status: new_status)
```

Comment:
Similar comment here (and for the calls to `create` in this service) about using the bang version of these DB methods (`update!` in this case). I can't think of a reason why the save would fail, but still feels safer to me.